src/askvm/resources/core/at.ts

```diff
@@ -3,6 +3,27 @@ import { any, resource, typed } from '../../lib';
 export const at = resource({
   type: any,
   async resolver(listOrObject: any, key: any): Promise<any> {
+    if (
+      Array.isArray(listOrObject) &&
+      Number.isInteger(key) &&
+      (listOrObject.length <= key || key < 0)
+    )
+      throw Error(
+        'Sorry, but index ' +
+          key +
+          ' is out of bounds for a list with ' +
+          listOrObject.length +
+          ' elements.'
```

Comment:
Can we use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here for greater visibility?

Reply:
sure 🙂

Reply:
@mhagmajer btw thanks for the direct link to documentation about template literals 😄